README.md

```diff
@@ -53,10 +53,31 @@ A DataProvider encaupsulates you data. Use one of the given DataProviders or imp
 ### ArrayDataProvider
 `ArrayDataProvider<Element>` wraps `Array<Element>` to a DataProvider.
 
+#### Initialize 
 ```swift
 let trains: [[Train]] = //
-let dataProvider = ArrayDataProvider(sections: trains, sectionIndexTitles: ["German", "French"])
+let dataProvider = ArrayDataProvider(sections: trains)
 ```
+ 
+#### Changing the data
+ ```swift
+ let newTrains: [[Train]] = //
+dataProvider.reconfigure(newTrains)
+ ```
+ If you calculate a diff for your new data, you can get insert/reorder/delete animation from your datasource.
```

Comment:
You can also provide cell animations if you process the differences – for e.g. 
let deletedIndexPath = IndexPath(row: 2, section: 0)
dataProvider.reconfigure(with: newTrains, updates: [.delete(deletedIndexPath)])